sqlengine/mysql_engine.go

```diff
@@ -34,6 +34,11 @@ func (d *MySQLEngine) Open(address string, port int64, dbname string, username s
 	return nil
 }
 
+func (d *MySQLEngine) Ping() error {
+	panic("Not implemented")
```

Comment:
Because that would hide the error. You would think that it pings successfully when it fact it does nothing.